src/actions/cmsActions.js

```diff
@@ -0,0 +1,10 @@
+import { createAsyncAction } from '../lib/reduxHelpers';
+import * as api from '../lib/cmsApi/rest';
```

Comment:
I don't think `import * as something` is a good idea in Javascript because it leads to namespace pollution and also  tree-shaking becomes less effective because you are importing everything from the module, whether you need it or not. It increases the bundle size unnecessarily.

Reply:
This made sense to me since we would always import everything from `lib/cmsApi/rest` in our actionCreator, similarly done [here](https://github.com/CodeForAfrica/web-tools/blob/main/src/actions/platformActions.js)